apps/monitoring/version-reporter/helm-charts.yaml

```diff
@@ -0,0 +1,42 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: version-reporter-service-helmcharts
+  namespace: monitoring
+  labels:
+    app: version-reporter-service
```

Comment:
Should usually match app name unless u have reason for not to

Reply:
Ok, no reason not to in this case, would update, thanks